libs/application/core/src/lib/ApplicationTemplateHelper.ts

```diff
@@ -166,8 +166,11 @@ export class ApplicationTemplateHelper<
     return returnValue
   }
   getWritableAnswersAndExternalData(
-    role: ApplicationRole,
+    role?: ApplicationRole,
   ): ReadWriteValues | undefined {
+    if (!role) {
```

Comment:
Same here with role, like with authentication - should this not just be assumed to be valid and caught elsewhere if not?